src/dlsnode/util/aio/internal/JobQueue.d

```diff
@@ -93,11 +93,11 @@ public static struct Job
 
     /****************************************************************
 
-        SuspendableRequestHandler used to wake the job.
+        ContextWaitingForJob used to wake the job.
 
     ****************************************************************/
 
-    public SuspendableRequestHandler suspendable_request_handler;
+    public ContextWaitingForJob context_waiting;
```

Comment:
I'd call this field `waiting_context` -- leveraging natural grammatical features of English (FTW).

Ditto for all variables with this name.

Reply:
FTW